.github/workflows/locale.yml

```diff
@@ -270,4 +270,23 @@ jobs:
         run: |
           make check
           if grep "DEPRECATED" Testing/Temporary/LastTest.log; then exit 1; fi
-
+  linux-bn-in:
+    name: "linux, bn-in"
+    runs-on: "ubuntu-20.04"
+    env:
+      LANG: "bn_IN.UTF-8"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Configure
+        run: |
+          cmake .
+      - name: Check Language
+        run: |
+          grep -q "STUMPLESS_LANGUAGE \"bn_IN\"" include/stumpless/config.h
```

Comment:
changing the underscore to a dash should fix this

Reply:
Fixed the typo error, have a look @goatshriek